tms/tests/test_fleet_vehicle.py

```diff
@@ -0,0 +1,54 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestFleetVehicle(TransactionCase):
```

Comment:
You should use `BaseCase` everywhere when tracking is not needed.
`setUp` code should be moved to `setUpClass` unless is really needed to run something before every test runs.